packages/squiggle-lang/src/rescript/Distributions/DistributionTypes.res

```diff
@@ -9,9 +9,43 @@ type error =
   | NotYetImplemented
   | Unreachable
   | DistributionVerticalShiftIsInvalid
+  | TooFewSamples
   | ArgumentError(string)
+  | OperationError(Operation.invalidOperationError)
+  | PointSetConversionError(SampleSetDist.pointsetConversionError)
+  | SparklineError(PointSetDist.sparklineError) // This type of error is for when we find a sparkline of a discrete distribution. This should probably at some point be actually implemented
   | Other(string)
```

Comment:
I think I'd like `OtherError`, frankly?

Reply:
(No preference on my side)

Reply:
This is tracked in https://github.com/quantified-uncertainty/squiggle/issues/327

Reply:
Mild preference for OtherError on my end. Will change